content/zh-cn/blog/_posts/2022-09-07-iptables-chains.md

```diff
@@ -0,0 +1,343 @@
+用作服务代理的一部分。 过去，kubelet 还使用 iptables
+来实现一些功能（例如为 Pod 设置 `hostPort` 映射），因此它也冗余地创建了一些重复的链。
+
+<!--
+However, with [the removal of dockershim] in Kubernetes in 1.24,
+kubelet now no longer ever uses any iptables rules for its own
+purposes; the things that it used to use iptables for are now always
+the responsibility of the container runtime or the network plugin, and
+there is no reason for kubelet to be creating any iptables rules.
+
+Meanwhile, although `iptables` is still the default kube-proxy backend
+on Linux, it is unlikely to remain the default forever, since the
+associated command-line tools and kernel APIs are essentially
+deprecated, and no longer receiving improvements. (RHEL 9
+[logs a warning] if you use the iptables API, even via
+`iptables-nft`.)
+-->
+然而，随着 1.24 版本 Kubernetes 中 [dockershim 的移除]，
+kubelet 现在不再为某种目的使用任何 iptables 规则；
+它过去使用 iptables 的事情现在总是由容器运行时或网络插件负责的，
+现在 kubelet 没有理由创建任何 iptables 规则。
+
+同时，虽然 iptables 仍然是 Linux 上默认的 kube-proxy 后端，
+但它不会永远保持默认，因为相关的命令行工具和内核 API 基本上已被弃用，
+并且不再得到改进。（RHEL 9 [记录警告] 如果你使用 iptables API，即使是通过 `iptables-nft`。）
+
+<!--
+Although as of Kubernetes 1.25 iptables kube-proxy remains popular,
+and kubelet continues to create the iptables rules that it
+historically created (despite no longer _using_ them), third party
+software cannot assume that core Kubernetes components will keep
+creating these rules in the future.
+
+[the removal of dockershim]: https://kubernetes.io/blog/2022/02/17/dockershim-faq/
+[logs a warning]: https://access.redhat.com/solutions/6739041
+-->
+尽管在 Kubernetes 1.25，iptables kube-proxy 仍然很流行，
+并且 kubelet 继续创建它过去创建的 iptables 规则（尽管不再**使用**它们），
+第三方软件不能假设核心 Kubernetes 组件将来会继续创建这些规则。
+
+[移除dockershim]: https://kubernetes.io/blog/2022/02/17/dockershim-faq/
+[记录警告]: https://access.redhat.com/solutions/6739041
+
+<!--
+## Upcoming changes
+
+Starting a few releases from now, kubelet will no longer create the
+following iptables chains in the `nat` table:
+
+  - `KUBE-MARK-DROP`
+  - `KUBE-MARK-MASQ`
+  - `KUBE-POSTROUTING`
+
+Additionally, the `KUBE-FIREWALL` chain in the `filter` table will no
+longer have the functionality currently associated with
+`KUBE-MARK-DROP` (and it may eventually go away entirely).
+
+This change will be phased in via the `IPTablesOwnershipCleanup`
+feature gate.  That feature gate is available and can be manually
+enabled for testing in Kubernetes 1.25. The current plan is that it
+will become enabled-by-default in Kubernetes 1.27, though this may be
+delayed to a later release. (It will not happen sooner than Kubernetes
+1.27.)
+-->
+## 即将发生的变化
+
+从现在开始的几个版本中，kubelet 将不再在 `nat` 表中创建以下 iptables 链：
+
+  - `KUBE-MARK-DROP`
+  - `KUBE-MARK-MASQ`
+  - `KUBE-POSTROUTING`
+
+此外，`filter` 表中的 `KUBE-FIREWALL` 链将不再具有当前与
+`KUBE-MARK-DROP` 关联的功能（并且它最终可能会完全消失）。
+
+此更改将通过 `IPTablesOwnershipCleanup` 特性门控逐步实施。
+可以手动在 Kubernetes 1.25 开启此特性中进行测试。
+目前的计划是将其在 Kubernetes 1.27 中默认启用，
+尽管这可能会延迟到以后的版本。（不会在 Kubernetes 1.27 版本之前调整。）
```

Comment:
Line 116 - 151 is too long. Please divide it into two or three segments, to provide convenience for reviewers.